gmcs/choices.py

```diff
@@ -2292,6 +2328,31 @@ def convert_31_to_32(self):
             if strat.get('possessum-type')=='non-affix':
                 if not strat.get('possessum-mark-order'):
                     strat['possessum-mark-order'] = hc
+
+    def convert_32_to_33(self):
+        pass
+        if self.get('q-part') == 'on':
+            orth = self.get('q-part-orth')
+            self.delete('q-part-orth')
+            self['q-particle'] = ChoiceList(full_key='q-particle')
+            cdict = ChoiceDict(full_key = 'q-particle1')
```

Comment:
What do you mean by "look wrong"?

Reply:
In #472 (still a draft) I removed the `full_keys()` method but not the `full_key` initialization parameter, but I expect it, too, will go away in the end.

Style-wise the spaces around the `=` are not PEP-8 compliant (but so far I haven't been raising style issues because there are so many in the codebase already... these could be fixed in a style-fixes-only kind of PR).

Reply:
@olzama Oh, now that I look at it again, I think I misunderstood what these were doing. These are top level values, so I guess they're correct.

@goodmami in the Choices redo (still not published...) that I've been working on, I have kept `full_key`. I don't think it's inherently bad (though sometimes used poorly), and it is used *a lot* in the code.

Reply:
@dantiston Ok I did `git grep full_key` and yes, it's more prevalent than I thought. Consider merging #472 to get rid of `full_keys()` method, which was only (ab)used in the adnominal_possession library. I still think most uses of `full_key` are not great. I was hoping we'd just use TOML and use the decoded dict instead of a custom object, but if we're using `full_key` it might not be that simple.